scene/gui/item_list.cpp

```diff
@@ -767,6 +769,8 @@ void ItemList::_gui_input(const Ref<InputEvent> &p_event) {
 
 		scroll_bar->set_value(scroll_bar->get_value() + scroll_bar->get_page() * pan_gesture->get_delta().y / 8);
 	}
+
+	if (scroll_bar->get_value() != prev_scroll) accept_event(); //accept event if scroll changed
```

Comment:
It would be more readable with a line break and indentation before `accept_event()` IMO.

Reply:
Ok, changed.